examples/rbfe_edge_list.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import csv
+import pickle
+
+import numpy as np
+from rdkit import Chem
+
+Chem.SetDefaultPickleProperties(Chem.PropertyPickleOptions.AllProps)
+
+from timemachine.fe import atom_mapping
+from timemachine.fe.rbfe import HostConfig, estimate_relative_free_energy
+from timemachine.ff import Forcefield
+from timemachine.md import builders
+from timemachine.parallel.client import CUDAPoolClient
+
+
+def get_mol_by_name(mols, name):
+    for m in mols:
+        if m.GetProp("_Name") == name:
+            return m
+
+    assert 0, "Mol not found"
+
+
+def run_pair(mol_a, mol_b, core, forcefield_path, protein_path, n_frames):
+
+    box_width = 4.0
+    solvent_sys, solvent_conf, solvent_box, solvent_top = builders.build_water_system(box_width)
+    solvent_box += np.diag([0.1, 0.1, 0.1])  # remove any possible clashes
```

Comment:
deboggle: seems like build_water_system should do this

Reply:
agreed - I think a recent version of OpenMM may have fixed this: https://github.com/openmm/openmm/issues/3502 but we need to validate